lib/Stripe.php

```diff
@@ -16,6 +16,9 @@ class Stripe
     // @var string|null The version of the Stripe API to use for requests.
     public static $apiVersion = null;
 
+    // @var string|null The account id for managed account requests.
```

Comment:
Sorry for the chunder...this is not managed accounts only. This should be `connected account`. It also works with your own account ID, but the purpose is connected accounts, whether managed or standalone.